createdisk.sh

```diff
@@ -206,6 +210,10 @@ if ! which ${VIRT_RESIZE}; then
     sudo yum -y install /usr/bin/virt-resize libguestfs-xfs
 fi
 
+if ! which ${VIRT_FILESYSTEMS}; then
+    sudo yum -y install /usr/bin/virt-filesystems
+fi
+
```

Comment:
It's in the same package as virt-resize, so most of the time this will be skipped.